identifiers/country-mx/first_level.csv

```diff
@@ -1,5 +1,10 @@
 id,name
 ocd-division/country:mx,Mexico
+ocd-division/country:mx/region:1,Mexico first region
```

Comment:
Should these be in Spanish? Also, "Mexico" and "Region" are redundant. If Spanish, use:

Primera
Segunda
Tercera
Cuarta
Quinta

Reply:
Done